purecharity-wp-fundraisers/purecharity-wp-fundraisers.php

```diff
@@ -110,11 +110,12 @@ function fr_force_template() {
 add_action( 'init', 'purecharity_wp_fundraisers_updater' );
 function purecharity_wp_fundraisers_updater() {
   if ( is_admin() ) { // note the use of is_admin() to double check that this is happening in the admin
-    $config = array(
+    $fr_config = array(
       'slug' => plugin_basename( __FILE__ ),
       'proper_folder_name' => 'purecharity-wp-fundraisers',
       'api_url' => 'https://api.github.com/repos/purecharity/pure-fundraisers',
       'raw_url' => 'https://raw.github.com/purecharity/pure-fundraisers/master',
+      'raw_url' => 'https://raw.githubusercontent.com/purecharity/pure-fundraisers/master/purecharity-wp-fundraisers/',
```

Comment:
I'm not sure if this is intentional but there are two entries for `raw_url` here